Core.Telemetry/OpenTelemetryMetric.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using OpenTelemetry;
+using OpenTelemetry.Exporter.Geneva;
+using OpenTelemetry.Metrics;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.CloudMine.Core.Telemetry
+{
+    public class OpenTelemetryMetric
+    {
+        public static TelemetryMetric<long> RecordCounter;
+        public static TelemetryMetric<long> HeartbeatCounter;
+        public static TelemetryMetric<double> RateLimitDelay;
```

Comment:
I did this to prevent devs from accidentally overlapping metric names. If two people were to add metrics with the same name (maybe: `Request`) we would start running into problems. Although assuming that we are checking in Geneva and in the PR process then maybe we do not need to do this.